src/Tribe/PUE/Checker.php

```diff
@@ -524,21 +510,28 @@ public function get_pue_update_url() {
 		}
 
 		/**
-		 * Get the plugin file path
+		 * Get the plugin file path.
 		 *
 		 * @return string
 		 */
-		public function get_plugin_file() {
+		public function get_plugin_file(): string {
+			/**
+			 * Filters the plugin file path for the current plugin.
+			 *
+			 * @since TBD
+			 *
+			 * @param string $plugin_file The default plugin file path.
+			 * @param string $slug        The slug of the current plugin.
+			 */
 			return apply_filters( 'pue_get_plugin_file', $this->plugin_file, $this->get_slug() );
```

Comment:
Going along with @bordoni 's suggestions above, we should probably update this line, too.
```suggestion
			return (string) apply_filters( 'pue_get_plugin_file', $this->plugin_file, $this->get_slug() );
```